frontend/src/metabase/search/components/InfoText/InfoTextAssetLink.tsx

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable react/prop-types */
+import { t } from "ttag";
+import { useDatabaseQuery, useTableQuery } from "metabase/common/hooks";
+import { Icon } from "metabase/core/components/Icon";
+import {
+  browseDatabase,
+  browseSchema,
+  tableRowsQuery,
+} from "metabase/lib/urls";
+import { SearchResultLink } from "metabase/search/components/SearchResultLink";
+import type { WrappedResult } from "metabase/search/types";
+import { Box, Text } from "metabase/ui";
+import type Database from "metabase-lib/metadata/Database";
+import { getInfoText } from "./get-info-text";
+import type { InfoTextData } from "./get-info-text";
+
+type InfoTextAssetLinkProps = {
+  result: WrappedResult;
+  isEnabled?: boolean;
+};
+
+const LinkSeparator = (
+  <Box component="span" c="text.1">
+    <Icon name="chevronright" size={8} />
+  </Box>
+);
+
+const LoadingText = ({ "data-testid": dataTestId = "loading-text" }) => (
+  <Text
+    color="text-1"
+    span
+    size="sm"
+    truncate
+    data-testid={dataTestId}
+  >{t`Loading…`}</Text>
+);
+
+export const InfoTextTableLink = ({
+  result,
+  isEnabled,
+}: InfoTextAssetLinkProps) => {
+  const { data: table, isLoading } = useTableQuery({
+    id: result.table_id,
+  });
+
+  const link = tableRowsQuery(result.database_id, result.table_id);
+  const label = table?.display_name ?? null;
+
+  if (isLoading) {
+    return <LoadingText data-testid="info-text-asset-link-loading-text" />;
```

Comment:
We don't need the `data-testId` prop now, we can just inline it in `LoadingText` definition.